style.css

```diff
@@ -0,0 +1,214 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+
+body {
+  width: 372px;
+  background-color: #f7f7f9;
+}
+
+.main {
+  background-color: white;
+  position: relative;
+  top: 0;
+  width: 375px;
+  background-image: url(header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  background-position: top;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.logo {
+  font-family: 'Poppins', sans-serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #6070ff;
+}
+
+.header {
+  font-family: 'Poppins', sans-serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: left;
+  margin-left: 20px;
+  margin-right: 20px;
+  padding-top: 100px;
+  color: #172b4d;
+}
```

Comment:
- [ ]  You have done a great job matching all sections with Figma template, however, the header section is slightly different: 

Figma Header  | Your Header
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/143862245-40266e70-345e-4863-8d37-c9eb583eb58d.png)  | ![image](https://user-images.githubusercontent.com/77038610/143862217-257e3fe8-7b38-43eb-b30d-2d9853b1062c.png)

Setting background color to white would solve the issue 👍🏻